lib/tasks/person_populate.rake

```diff
@@ -0,0 +1,13 @@
+# TODO: Remove this once we've migrated all profile attributes to a person and are updating these attributes dynamically
+namespace :person do
+  desc 'Populates a person with profile field values'
+  task populate: :environment do
+    total = Person.count
+
+    Person.find_each do |person|
+      PersonPopulator.new(person, person.latest_profile).call
```

Comment:
It should be nice to some output here,
since there is not any output, we do not have any idea on the progression, 
this could take tens of min minutes.
And also we do not want too much output :)
You could use the trick

```
.find_each(batch_size: 200).with_index do |move, n|
      puts "#{n}/#{total} person processed ..." if (n % 200).zero? 

```
(see move_profile.rake)

Reply:
Done :)